sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2303,6 +2306,14 @@ export { IHttpClient }
 // @public
 export function isPipelineLike(pipeline: unknown): pipeline is PipelineLike;
 
+// @public
+export const enum KnownBlobAudience {
```

Comment:
There's discussion and recommendation to remove "Known" prefix. I am thinking about "StorageBlobAudience" because "BlobAudience" doesn't sound as clear enough. @xirzec @bterlson any suggestion?

Reply:
Updated to StorageBlobAudience and removed BlobAudience type.